src/Pages/check/check.tsx

```diff
@@ -1,10 +1,13 @@
-import React from "react";
+import { Navbar } from "../../components";
 
 const check = () => {
   return (
-    <div className="text-3xl w-screen h-screen flex items-center justify-center bg-violet-600">
-      Check flights page
-    </div>
+    <>
+      <Navbar />
```

Comment:
Curious: `<Navbar />` seems to be included in a lot of pages. If the props for this were ever needed to change it would need updates in n places to apply the props.

- If it's on every page is there one base place we could include it so it doesn't need to be included on every page?
- If it's not on every single page, or it's different on a per page basis, is there a way you could extract it so it's reusable by pages?

For example, you could extra it and have a `layouts` folder that included layouts, `<StandardPageLayout>` that could wrap standard pages and include layout, and hypothetically say there was a dashboard layout that didn't have navbar you could have a component for that.

Little extractions like this are a huge win for reducing duplication and easing maintenance by reducing the number of changes needed for updates.

Reply:
So your pages might look something like this:

```
src/layouts/StandardLayout.tsx

const StandardLayout = (props) => {
   <>
     <Navbar>
     // Challenge - how do you render react children through props? 
   </>
}

```
import { StandardLayout } from '../..//layouts'

const check = () => (
  <StandardLayout>
      <div>Check flights page</div>
  </StandardLayout>
);
```

Reply:
Aah as I'm writing this as well, a nit but highly recommended as a naming convention to get used too - React components get exported as CamelCase, both in the component and the filenames.

So `const check` would become `const Check` and `check.tsx` would become `Check.tsx`.

This should happen across all react components regardless of type.